src/main/scala/cognite/spark/v1/FlexibleDataModelCorePropertySyncRelation.scala

```diff
@@ -36,8 +37,11 @@ private[spark] class FlexibleDataModelCorePropertySyncRelation(
         corePropConfig.instanceSpace)
     )(sqlContext) {
 
+  private val logger = getLogger
   // request no more data after seeing items with node/edge lastUpdatedTime >= terminationTimeStamp
   private val terminationTimeStamp = System.currentTimeMillis()
+  private val mandatoryViewFields = schema.fields.filter(f =>
+    f.name != "space" && f.name != "externalId" && f.name != "type" && f.name != "startNode" && f.name != "endNode" && !f.nullable)
```

Comment:
isn't space,externalId mandatory?
Maybe should be called `requiredViewProperties` and a comment that it doesn't include system/non-customizable

Reply:
right, need to node include node/edge properties here, only view properties.. 
can rename and comment